packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -48,6 +48,29 @@ enum DragStartBehavior {
   start,
 }
 
+/// Configuration of multi-finger drag strategy on the multi-touch devices.
+///
+/// When dragging with only one finger, there's no difference in behavior
+/// between the two settings.
+///
+/// Used by [DragGestureRecognizer.multitouchDragStrategy].
+enum MultitouchDragStrategy {
```

Comment:
We should probably add another option that allows the user to revert to the old Flutter behavior, just in case anyone wants it.

Reply:
That is one of the options here.

Reply:
I see, I thought `trackAllActivePointers` referred to the iOS-style behavior (and hence my other misunderstanding). So if we decides to go with this name, how do we plan to name the iOS-style behavior?

Maybe rename it to `sumAllPointers` (for the old behavior) and `averageAllPointers` (for the iOS-style)?

Also, do we need the "active" word here? It doesn't seem needed to me here and I'd prefer shorter names if possible.